esports_data_from_pandascore/fn-call.json

```diff
@@ -2,10 +2,10 @@
   "code_file": "tmp/x.wasm",
   "function": "scoreFunc",
   "input": {
-    "matches_api_endpoint": "PandaScore API matches endpoint",
+    "matches_api_endpoint": "{{ .YOUR_PANDASCORE_API_ENDPOINT }}",
     "match_id": "1121861"
   },
   "secret": {
-    "api_key": "PandaScore API Key"
+    "api_key": "{{ .YOUR_PANDASCORE_API_KEY }}"
```

Comment:
for the examples where you have modified files in the example (pandascore, dhl, sportsradar, rimble) can you check the readmes in those examples and update any relevant parts of the readmes to reflect your changes? There are likely parts of the readmes that instruct users on what strings to replace with their own values. Similar comment/request here as an example: https://github.com/blocky/attestation-service-examples/pull/48#discussion_r2089805701

Reply:
done